lib/ayesql/compiler.ex

```diff
@@ -181,7 +189,47 @@ defmodule AyeSQL.Compiler do
   end
 
   @spec create_query(query()) :: Macro.t()
-  defp create_query({name, docs, fragments}) do
+  defp create_query({name, :script, docs, fragments}) do
```

Comment:
@dkuku There are several reasons why I don't think modifying the parser is a good approach (see my following comments):